BMM.UI.Android/Application/Fragments/PlayerFragment.cs

```diff
@@ -31,6 +32,8 @@ namespace BMM.UI.Droid.Application.Fragments
     [Register("bmm.ui.droid.application.fragments.PlayerFragment")]
     public class PlayerFragment : BaseFragment<PlayerViewModel>, SeekBar.IOnSeekBarChangeListener
     {
+        private const int TimeToCheckEmptyPlayerErrorInMillis = 5000;
```

Comment:
I think 5 seconds is too long. The user would actually need to stare at the player for that long. 1 second should be better

Reply:
Not exactly "stare". The player always opens as PlayerFragment, event if it's only mini player at the bottom, so the user can freely navigate over the app, but this fragment is still shown.  
I think I'll change it to 2 seconds.

Reply:
I see. That makes sense